src/AddOrder.js

```diff
@@ -1,200 +1,373 @@
+                  <Form.Control
+                    required
+                    type="text"
+                    name="firstName"
+                    placeholder="Imię"
+                    isInvalid={errors.firstName}
+                    isValid={isSubmitted && !errors.firstName}
+                    {...register("firstName", {
+                      required: "Wymagane",
+                    })}
+                  />
+                  <Form.Control.Feedback type="invalid">
+                    {errors.firstName?.message}
+                  </Form.Control.Feedback>
+                </FloatingLabel>
+              </Col>
+              <Col>
+                <FloatingLabel label="Nazwisko" controlId="formLastName">
+                  <Form.Control
+                    required
+                    type="text"
+                    name="lastName"
+                    placeholder="Nazwisko"
+                    isInvalid={errors.lastName}
+                    isValid={isSubmitted && !errors.lastName}
+                    {...register("lastName", {
+                      required: "Wymagane",
+                    })}
+                  />
+                  <Form.Control.Feedback type="invalid">
+                    {errors.lastName?.message}
+                  </Form.Control.Feedback>
+                </FloatingLabel>
+              </Col>
+            </Row>
+            <Row className="g-2 mt-1">
+              <Col>
+                <FloatingLabel label="Nr. Telefonu" controlId="formPhoneNumber">
+                  <Form.Control
+                    required
+                    type="tel"
+                    name="phoneNumber"
+                    placeholder="Nr. Telefonu"
+                    isInvalid={errors.phoneNumber}
+                    isValid={isSubmitted && !errors.phoneNumber}
+                    {...register("phoneNumber", {
+                      required: "Wymagane",
+                      pattern: {
+                        value: /^[\\+]?[(]?[0-9]{3}[)]?[-\s\\.]?[0-9]{3}[-\s\\.]?[0-9]{4,6}$/,
+                        message: "Niepoprawny numer telefonu",
+                      }
+                    })}
+                  />
+                  <Form.Control.Feedback type="invalid">
+                    {errors.phoneNumber?.message}
+                  </Form.Control.Feedback>
+                </FloatingLabel>
+              </Col>
+              <Col>
+                <FloatingLabel label="E-mail" controlId="formEmail">
+                  <Form.Control
+                    required
+                    type="email"
+                    name="email"
+                    placeholder="E-mail"
+                    isInvalid={errors.email}
+                    isValid={isSubmitted && !errors.email}
+                    {...register("email", {
+                      required: "Wymagane",
+                      pattern: {
+                        value: /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/,
```

Comment:
Górny limit TLD 4 znaki nie może być:
https://en.wikipedia.org/wiki/List_of_Internet_top-level_domains#ICANN

Reply:
daj mi lepszy regex jak znasz to podmienię

Reply:
```
^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$
```
https://www.abstractapi.com/tools/email-regex-guide